packages/console/src/pages/SignInExperience/components/GuideModal.tsx

```diff
@@ -73,17 +73,22 @@ const GuideModal = ({ isOpen, onClose }: Props) => {
     onClose();
   });
 
+  const onSkip = async () => {
+    await updateSettings({ customizeSignInExperience: true });
```

Comment:
if we want to await should we add a loading state? or we can just use `void` on this.

Reply:
agreed, would there be a multi-click lock?

Reply:
Oh, loading state is needed, updated.